app/models/manager_refresh/inventory_collection_default/infra_manager.rb

```diff
@@ -122,55 +208,81 @@ def ems_clusters(extra_attributes = {})
     def storages(extra_attributes = {})
       attributes = {
         :model_class => ::Storage,
-        :association => :storages,
+        # TODO: change :manager_ref => [:location],
+        :association => :storages
```

Comment:
Correct, I wanted to use this code but for `ovirt-image-repository` `                   storage_inv.attributes.fetch_path(:storage, :volume_group, :logical_unit, :id)` returns `nil`. We always have such storage per provider which makes it not unique. For this specific storage I could use `:ems_ref` but not sure whether it would break anything.